moveit_setup_assistant/src/tools/collision_linear_model.cpp

```diff
@@ -234,10 +237,15 @@ bool SortFilterProxyModel::filterAcceptsRow(int source_row, const QModelIndex& s
   CollisionLinearModel* m = qobject_cast<CollisionLinearModel*>(sourceModel());
   if (!(show_all_ || m->data(m->index(source_row, 2), Qt::CheckStateRole) == Qt::Checked))
     return false;  // not accepted due to check state
-
+#if (QT_VERSION < QT_VERSION_CHECK(5, 12, 0))
```

Comment:
QRegularExpression is available since Qt5.

Reply:
Yes, but [filterRegularExpression](https://doc.qt.io/qt-5/qsortfilterproxymodel.html#filterRegularExpression-prop) as a replacement for `filterRegExp` was only introduced in Qt 5.12